server/ai/subtitles/subtitles.go

```diff
@@ -13,6 +14,38 @@ type Subtitles struct {
 	storage *astisub.Subtitles
 }
 
+func readChat(chat io.Reader) (*astisub.Subtitles, error) {
+	storage := astisub.NewSubtitles()
+
+	scanner := bufio.NewScanner(chat)
+	for scanner.Scan() {
+		line := scanner.Text()
+		text := line[9:]
+		item := &astisub.Item{}
+		startAt, err := time.Parse("15:04:05", line[:8])
+		if err != nil {
+			return nil, err
+		}
+		zeroTime, err := time.Parse("15:04:05", "00:00:00")
+		if err != nil {
+			return nil, err
+		}
+		item.StartAt = startAt.Sub(zeroTime)
+		item.EndAt = startAt.Add(5 * time.Second).Sub(zeroTime)
+		item.Lines = append(item.Lines, astisub.Line{Items: []astisub.LineItem{{Text: text}}})
+		storage.Items = append(storage.Items, item)
+	}
+	return storage, nil
+}
+
+func NewSubtitlesFromChat(chat io.Reader) (*Subtitles, error) {
```

Comment:
Include zoom here because it's a zoom specific format. `NewSubtitlesFromZoomChat` same for above helper function.